webview-ui/src/i18n/locales/en/kilocode.json

```diff
@@ -44,5 +44,36 @@
 			"description": "Signing up with Kilo Code gives you free credits to get started. Use your credits to explore our features, try out the latest and greatest models, and experience the benefits of Kilo Code without any commitment.",
 			"termsAndPrivacy": "By continuing, you agree to the <termsLink>Terms of Service</termsLink> and <privacyLink>Privacy Policy.</privacyLink>"
 		}
+	},
+	"docs": "Docs",
+	"rules": {
+		"tooltip": "Manage Kilo Code Rules & Workflows",
+		"ariaLabel": "Kilo Code Rules",
+		"tabs": {
+			"rules": "Rules",
+			"workflows": "Workflows"
+		},
+		"description": {
+			"rules": "Rules allow you to provide Kilo Code with system-level guidance. Think of them as a persistent way to include context and preferences for your projects or globally for every conversation.",
```

Comment:
```suggestion
			"rules": "Rules allow you to provide Kilo Code with instructions it should follow in all modes and for all prompts. They are a persistent way to include context and preferences for all conversations in your workspace or globally.",
```

Reply:
is that slightly less wishy-washy?  Not a fan of the term "guidance" in the old text.